lib/modem_slm/Kconfig

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MODEM_SLM
+	depends on SERIAL
+	depends on UART_ASYNC_API
+	depends on RING_BUFFER
+	select EXPERIMENTAL
+	bool "Modem SLM Library for MCU"
```

Comment:
We usually define the config type and description as the first item. Please also add `[EXPERIMENTAL]` at the end of the description so that it's clearly visible in the menuconfig.